release-notes/v0.3.0.yaml

```diff
@@ -0,0 +1,81 @@
+date: February 09, 2023
+
+changes:
+  - area: documentation
+    change: |
+      Added Global Rate Limit User Docs
+      Added Request Authentication User Docs
+      Added TCP Routing User Docs
+      Added UDP Routing User Docs
+      Added GRPC Routing User Docs
+      Added HTTP Response Headers User Docs
+      Added TCP and UDP Proxy Design Docs
+      Added egctl Design Docs
+      Added Rate Limit Design Docs
+      Added Request Authentication Design Docs
+      Added Support for Versioned Docs
+      Added Support for Multiple Release Versions
+      Added Release Details Docs
+      Added API Docs Generating Tooling
+      Refactored Layout for User Docs
+
+  - area: api
+    change: |
+      Upgraded to v0.6.1 Gateway API
+      Added Support for the TCPRoute API
+      Added Support for the UDPRoute API
+      Added Support for the GRPCRoute API
+      Added Support for HTTPRoute URLRewrite Filter
+      Added Support for HTTPRoute RequestMirror Filter
+      Added Support for HTTPRoute ResponseHeaderModifier Filter
+      Added Support for Request Authentication 
+      Added Support for Global Rate Limiting
+      Added Support for Routes ReferenceGrant
+      Added Support for Namespace Server Config Type
+      Added initial management of Envoy Proxy deployment via EnvoyProxy API
+
+  - area: ci-tooling-testing
+    change: |
+      Fixed Make Image Failed in Darwin
+      Fixed Wait for Job Succeeded before conformance test
+      Upgraded Echoserver Image Tag
+      Added Support for User-Facing Version
+      Added Support for Testing EG against Multiple Kubernetes Versions
+
+  - area: conformance
+    change: |
+      Enabled GatewayClassObservedGenerationBump conformance test
```

Comment:
Remove since the test was disabled today due to flakes.

Reply:
this one wasn't @danehans

Reply:
the flakes were for `HTTPRouteObservedGenerationBump` and `GatewayObservedGenerationBump`